src/extension.ts

```diff
@@ -176,6 +177,8 @@ export async function activate(context: vscode.ExtensionContext) {
             toolkitSettings
         })
 
+        activateAwsSamDebugger(context)
```

Comment:
Can we add this activation into /src/shared/sam/activation.ts instead of creating a new one? That is where the existing sam functionality is activated.

Reply:
I'm a little conflicted, would we want to consider this SAM functionality or debugger functionality (i.e. will we add more debuggers in the future)? FWIW the other code I've written is in `/src/debugger`; should that get moved to SAM as well?

Reply:
All of the code produced in this change is SAM specific debugger functionality. There isn't anything general here. If we added debugging support for some other construct, it too would belong under the domain of that construct.

Now that you mention it, I would place the debug related files in a `debugger` subfolder _under_ the sam folder, for clear grouping within SAM (just like the cli code is under sam/cli).

Digression: /src/shared/sam should move to /src/sam, but that is separate from this.